veps/sig-compute/decentralized-migration/decentralized-migration-vep.md

```diff
@@ -163,14 +163,15 @@ metadata:
   name: vmim-target
 spec:
   receive:
-    key: <unique id>
+    migrationId: <unique id>
   vmName: vmi-name
 status:
   syncEndpoint: <address:port>
 ```
 
 If neither `sendTo` or `receive` is specified it means this is a normal in namespace live migration, and the `VirtualMachineInstanceMigration` can be treated as if it is both the `source` and the `target` migration resource.
 
+The `synchronization endpoint` is available on the cluster level and can be set by either the cluster administrator or generated by kubevirt during startup of the `synchronization controller`. If deploying KubeVirt with the kubevirt operator, it makes sense to display the address and port in the status of the KubeVirt resource.
```

Comment:
Question, can we have multiple endpoints? Like migrating to different cluster at the same time? Of course for different VMs.